share/functions/__fish_print_xdg_mimetypes.fish

```diff
@@ -1,5 +1,3 @@
 function __fish_print_xdg_mimetypes --description 'Print XDG mime types'
-    cat ~/.local/share/applications/mimeinfo.cache | grep -v "\[MIME Cache\]" | tr = \t
-    cat /usr/share/applications/mimeinfo.cache | grep -v "\[MIME Cache\]" | tr = \t
-
+    sed -e '/\[MIME Cache\]/d; y!=!\t!' ~/.local/share/applications/mimeinfo.cache /usr/share/applications/mimeinfo.cache
```

Comment:
If we're rewriting this anyway, I'd rather have it use `string`. Some `sed`s are weird and can't handle e.g. a "\t" in the match-expression, so I'm wary of using it any more than we have to.